If-CI-checks-fail-on-your-PR.md

```diff
@@ -1,42 +1,71 @@
 ## Table of contents
 
-* [Introduction](#introduction)
+* [General instructions](#general-instructions)
+  * [Why is an unrelated CI check failing on my PR?](#why-is-an-unrelated-ci-check-failing-on-my-pr)
+  * [Failures in the merge queue](#failures-in-the-merge-queue)
 * [Figuring out whether the failure is due to your PR or an existing issue](#figuring-out-whether-the-failure-is-due-to-your-pr-or-an-existing-issue)
   * [What to do if the failure is due to your PR](#what-to-do-if-the-failure-is-due-to-your-pr)
   * [What to do if the failure is due to an existing issue](#what-to-do-if-the-failure-is-due-to-an-existing-issue)
-* [What to do if CI checks only fail in the merge queue](#what-to-do-if-ci-checks-only-fail-in-the-merge-queue)
 
-## Introduction
 
-If your PR build fails, do not despair! Scroll down to the bottom of the PR thread until you see the results of the continuous integration (CI) tests that ran:
+## General instructions
 
-  ![Screenshot of PR CI results](images/ghciSample.png)
+If your PR build fails, do not despair! Follow these instructions:
 
-In the example above, the lint checks and Mypy checks have failed. Also, two checks were skipped due to the failure of one of the e2e tests. (This is because, in the event of an e2e test failure, any remaining e2e tests that are queued or currently running will be terminated and marked as "skipped.") The rest of the tests have passed.
+1. Scroll down to the bottom of the PR thread until you see the results of the continuous integration (CI) tests that ran.
 
-To diagnose and fix a failing check on your PR, follow the instructions below:
+1. If you see a warning that your PR has a **merge conflict**, you will need to resolve the conflict. (If not, you can skip this step.) To resolve a merge conflict, follow the "merge from develop" instructions in  [step 5 of our PR guide](https://github.com/oppia/oppia/wiki/Rules-for-making-PRs#step-5-address-review-comments-until-all-reviewers-approve). Once you push the merge commit to your feature branch on GitHub, the merge conflict message will disappear.
 
-1. If you see a warning that your PR has a **merge conflict**, you will need to resolve the conflict. To do this, follow the "merge from develop" instructions in  [step 5 of our PR guide](https://github.com/oppia/oppia/wiki/Rules-for-making-PRs#step-5-address-review-comments-until-all-reviewers-approve). Once you push the merge commit to your feature branch on GitHub, the merge conflict message will disappear.
+1. At the bottom of the PR thread, you will see a set of results that looks similar to the following. (**Note:** If your failure happens in the merge queue, you might need to click on "View details" to see the corresponding table.)
```

Comment:
what's the "corresponding table"? If it's the table of results in the screenshot, how about "to see the results."?

Reply:
Done.